contracts/test/integration/vip/vip13.sol

```diff
@@ -0,0 +1,477 @@
+            assertEq(psmTargetBalance, targetBalanceUsdc);
+            assertEq(decimalsNormalizer, usdcDecimalNormalizer);
+        }
+        {
+            (
+                address psmToken,
+                uint248 psmTargetBalance,
+                int8 decimalsNormalizer
+            ) = allocator.allPSMs(address(voltV2DaiPriceBoundPSM));
+            assertEq(psmToken, MainnetAddresses.DAI);
+            assertEq(psmTargetBalance, targetBalanceDai);
+            assertEq(decimalsNormalizer, daiDecimalNormalizer);
+        }
+        /// assert that DAI PSM and USDC PSM are whitelisted in PCV Guardian
+        assertTrue(
+            PCVGuardian(MainnetAddresses.PCV_GUARDIAN).isWhitelistAddress(
+                address(voltV2UsdcPriceBoundPSM)
+            )
+        );
+
+        assertTrue(
+            PCVGuardian(MainnetAddresses.PCV_GUARDIAN).isWhitelistAddress(
+                address(voltV2DaiPriceBoundPSM)
+            )
+        );
+    }
+
+    /// prevent errors by reverting on arbitrum proposal functions being called on this VIP
+    function getArbitrumProposal()
+        public
+        pure
+        override
+        returns (ITimelockSimulation.action[] memory)
+    {
+        revert("no arbitrum proposal");
+    }
+
+    function arbitrumSetup() public override {
+        revert("no arbitrum proposal");
+    }
+
+    function arbitrumValidate() public override {
+        revert("no arbitrum proposal");
+    }
+
+    function deployPsms(address _voltV2) internal {
+        PegStabilityModule.OracleParams memory oracleParamsDai;
+        PegStabilityModule.OracleParams memory oracleParamsUsdc;
+
+        oracleParamsDai = PegStabilityModule.OracleParams({
+            coreAddress: address(MainnetAddresses.CORE),
+            oracleAddress: address(MainnetAddresses.ORACLE_PASS_THROUGH),
+            backupOracle: address(0),
+            decimalsNormalizer: 0,
+            doInvert: true,
+            volt: IVolt(_voltV2)
+        });
+
+        oracleParamsUsdc = PegStabilityModule.OracleParams({
+            coreAddress: address(MainnetAddresses.CORE),
+            oracleAddress: address(MainnetAddresses.ORACLE_PASS_THROUGH),
+            backupOracle: address(0),
+            decimalsNormalizer: 12,
+            doInvert: true,
+            volt: IVolt(_voltV2)
+        });
+
+        voltV2DaiPriceBoundPSM = new PriceBoundPSM(
+            voltDaiFloorPrice,
+            voltDaiCeilingPrice,
+            oracleParamsDai,
+            0,
+            0,
+            type(uint256).max,
+            10_000e18,
+            10_000_000e18,
+            IERC20(address(MainnetAddresses.DAI)),
+            IPCVDeposit(address(1))
+        );
+
+        voltV2UsdcPriceBoundPSM = new PriceBoundPSM(
+            voltUsdcFloorPrice,
+            voltUsdcCeilingPrice,
+            oracleParamsUsdc,
+            0,
+            0,
+            type(uint256).max,
+            10_000e18,
+            10_000_000e18,
```

Comment:
Would be good to add comments here that these three values are garbage values from a legacy system and don't affect anything.